timedistance-api.md

```diff
@@ -0,0 +1,82 @@
+
+# Valhalla Time Distance Service API Reference
+
+The time distance service is a free, open-source web API (and c++ library) that provides a quick computation of time and distance between a set of locations. This page documents the inputs and outputs to the service.
+
+The time distance service is in active development. You can follow the [Mapzen blog](https://mapzen.com/blog) to get updates. To report software issues or suggest enhancements, open an issue in the [Thor GitHub repository](https://github.com/valhalla/thor/issues). You can also send a message to [routing@mapzen.com](mailto:routing@mapzen.com).
+	
+#### API keys and Service Limits
+
+To use the Valhalla routing service, you must first obtain a free developer API key from Mapzen. Sign in at https://mapzen.com/developers to create and manage your API keys.
+
+Valhalla is a free, shared routing service. As such, there are limitations on requests, maximum distances, and numbers of locations to prevent individual users from degrading the overall system performance.
+
+The following time and distance limitations that are currently in place:
+
+* `one_to_many` and `many_to_one` requests have a limit of <?> locations.
+* `many_to_many` requests have a limit of <?> locations.
+* max distance between any two locations is <will these be the same as the limits on the route api reference page?>.
```

Comment:
TODO - we need to come up with some total distance limit - all locations are within X miles/km of each other or perhaps some centroid and radius limit. Not sure which is easiest to explain.